docs/sources/features/explore/index.md

```diff
@@ -200,6 +200,28 @@ While in Live tail view new logs will come from the bottom of the screen and wil
 
 {{< docs-imagebox img="/img/docs/v64/explore_live_tailing.gif" class="docs-image--no-shadow" caption="Explore Live tailing in action" >}}
 
+## Tracing integration
+> Only available in Grafana v7.0+.
+
+You can visualize traces from tracing data sources in explore. Data sources currently supported:
+- [Jaeger]({{< relref "../datasources/jaeger" >}})
+- [Zipkin]({{< relref "../datasources/zipkin" >}})
+
+For information about how to use the query editor see documentation for specific data source.
+
+TODO: image of the trace view
+
+There are two main sections of the tracing view: header and timeline.
+
+TODO: The header with zoom
+
+Header shows trace metadata like root span name, trace ID or start time. It also shows condensed minimap of all the spans. You can use this minimap to zoom to a section of trace timeline by dragging with your mouse and selecting a region. You can reset the zoom selection with Reset Selection button which is shown when you hover over the minimap. Use search to the right of the root span name to highlight any spans containing the search text.
```

Comment:
There is a LOT of information packed into these two paragraphs. Some of it is description, some of it looks like it is meant to be instruction. I recommend breaking it up and describing each part of the header and timeline and what you can do with it. Right now, the text is so dense I am dizzy reading it.

Reply:
To be honest not exactly sure how to describe this, the UI is bit complex but at the same time probably familiar to lots of people and having a picture will probably explain it better. I think the best for explaining UI interfaces would be an image of the interface with numbered and circled items and then some list of those items with a description underneath. Do you know if there is some tool for that, which would do that nicely without need to manually label things in photoshop or paint or something?